README.md

```diff
@@ -51,7 +51,18 @@ model = FSDP(model, use_orig_params=True)
 # optional: enable torch.compile for improved performance
 m = torch.compile(m)
 
-# train/finetune (not shown)
+# toy training loop
+for _ in range(N_ITER):
+    optimizer.zero_grad()
+    y = m(x)
+    y.sum().backward()
+    optimizer.step()
+
+    # specific to fsdp2 + float8 with dynamic scaling
```

Comment:
should we say that this is specific to FSDP2 with float8 all-gather turned on?  Also, maybe we can show how to turn that on, since I don't think it's documented in the README yet? Can be a follow-up PR.

Reply:
> should we say that this is specific to FSDP2 with float8 all-gather turned on?

will change it in this PR

> maybe we can show how to turn that on, since I don't think it's documented in the README yet

good catch. will polish README again after landing changes in torchtitan to turn on/off fp8 all-gather